cuda/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+cmake_minimum_required(VERSION 3.18 FATAL_ERROR)
```

Comment:
how do we guide Perlmutter user to use >= 3.22

Reply:
We could mention that NVHPC installs (and CUDA installations via NVIDIA HPC Toolkit) need CMake 3.22 in our README. Otherwise maybe a warning if the cuBLAS target is missing?

Reply:
At least ReadME, I think